frontend/src/components/dropdowns/YearDropdownField/index.js

```diff
@@ -15,12 +15,13 @@ const yearDropdownChoices = (minYear, maxYear) => {
 };
 
 const YearDropdownField = props => {
-  const { name, label, minYear, maxYear, ...otherProps } = props;
+  const { name, label, minYear, maxYear, fullWidth, ...otherProps } = props;
```

Comment:
Tbh you don't need to explicitly expose `fullWidth` since it will already be included in `otherProps`. I'm fine with you explicitly exposing it though so you need not remove it if you don't want to.